test/320_claim_3_test.sh

```diff
@@ -54,22 +68,25 @@ weave_on $HOST3 prime
 stop_weave_on $HOST2
 weave_on $HOST1 launch-router --ipalloc-range $UNIVERSE $HOST3
 
+docker_on $HOST1 stop c1
 stop_weave_on $HOST1
-stop_weave_on $HOST2
 stop_weave_on $HOST3
-delete_persistence $HOST1 $HOST2 $HOST3
+## Do not remove the state of host2 to keep the previously established ring.
```

Comment:
Not stopping host2 seems to invalidate the statement about "host2 being offline" below.